topics/transcriptomics/tutorials/scrna-intro/slides.html

```diff
@@ -0,0 +1,1069 @@
+---
+name: tutorial_slides
```

Comment:
this should be `layout: ` which could definitely be causing your rendering issues

Reply:
((screams internally))

thank you! I can likely proceed on my own again with this now

Reply:
no worries, our linter caught this but is showing a green checkmark when it shouldn't.. that woulda helped you debug, will look into fixing that